packages/dev/core/src/Engines/Native/nativeInterfaces.ts

```diff
@@ -99,10 +99,15 @@ export interface INativeEngine {
     submitCommands(): void;
 }
 
+/** @internal */
+interface INativeEngineFeatures {

```

Comment:
I would consider generalizing this to something like `VersionInfo` and include the BJS version number as a property as well.